app/src/main/java/org/covidwatch/android/ui/menu/MenuAdapter.kt

```diff
@@ -1,37 +1,109 @@
-        MenuItem(R.string.terms_of_use, R.drawable.ic_exit_to_app, Browser("https://www.covid-watch.org/")),
-        MenuItem(R.string.privacy_policy, R.drawable.ic_exit_to_app, Browser("https://www.covid-watch.org/"))
+        MenuItem(
+            R.string.generate_random_exposure,
+            R.drawable.ic_exit_to_app,
+            Browser("https://www.covid-watch.org/")
+        ),
+        MenuItem(
+            R.string.settings,
+            0,
+            Settings
+        ),
+        MenuItem(
+            R.string.possible_exposures,
+            R.drawable.ic_info_red,
+            TestResults),
+        MenuItem(
+            R.string.notify_others,
+            0,
+            Browser("https://www.covid-watch.org/")),
+        MenuItem(
+            R.string.how_the_app_works,
+            0,
+            Browser("https://www.covid-watch.org/")),
+        MenuItem(
+            R.string.health_guidelines,
+            R.drawable.ic_exit_to_app,
+            Browser("https://www.covid-watch.org/")
+        ),
+        MenuItem(
+            R.string.covid_watch_website,
+            R.drawable.ic_exit_to_app,
+            Browser("https://www.covid-watch.org/")
+        ),
+        MenuItem(
+            R.string.faq,
+            R.drawable.ic_exit_to_app,
+            Browser("https://www.covid-watch.org/")
+        ),
+        MenuItem(
+            R.string.terms_of_use,
+            R.drawable.ic_exit_to_app,
+            Browser("https://www.covid-watch.org/")
+        ),
+        MenuItem(
+            R.string.privacy_policy,
+            R.drawable.ic_exit_to_app,
+            Browser("https://www.covid-watch.org/")
+        ),
+        MenuItem(
+            R.string.get_support,
+            R.drawable.ic_exit_to_app,
+            Browser("https://www.covid-watch.org/")
+        )
     )
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MenuItemViewHolder {
         val root = LayoutInflater.from(parent.context).inflate(R.layout.item_menu, parent, false)
+        isHighRisk = getIsHighRisk(root.getContext())
         return MenuItemViewHolder(root)
     }
 
     override fun getItemCount(): Int = items.size
 
     override fun onBindViewHolder(holder: MenuItemViewHolder, position: Int) {
-        val menuItem = items[position]
+        var menuItem = items[position]
+        if (menuItem.title == R.string.possible_exposures) {
+            menuItem = updateMenuItemForRisk(menuItem)
+        }
+
         holder.bind(menuItem)
 
         holder.itemView.setOnClickListener {
             onClick(menuItem.destination)
         }
+
+    }
+
+    private fun updateMenuItemForRisk(menuItem: MenuItem): MenuItem {
+        if (isHighRisk) {
+            menuItem.iconEnd = R.drawable.ic_info_red
+        } else {
+            menuItem.iconEnd = 0
+        }
+        return menuItem
     }
+
+    private fun getIsHighRisk(context: Context): Boolean {
```

Comment:
The adapter class shouldn't handle business logic inside of it. It just binds data into UI.
`MenuFragment` should take care of this and change the items in the `MenuAdapter`.

The fragment should listen to the changes in `exposureSummary` as in `HomeViewModel` and change the `items` list in the `MenuAdapter`.